### PR TITLE
Doc/gh 353

### PR DESCRIPTION
**Pull request by shaun-09**
Customer doc for the Stock Notification feature is now ready for merge.

https://github.com/SAP/cloud-commerce-spartacus-storefront-docs/blob/doc/GH-353/_pages/dev/features/stock-notification.md 

Closes GH-353

### CODE REVIEW

**Comment by gladius-mtl** [comment]
Hi Shaun! I will approve this as soon as Stock Notification code is merged 🙂

**Comment by gladius-mtl** [comment]
Hi Shaun, I approved the PR to see if it would resolve a problem I have encountered on the develop branch -- but it did not, so I dismissed my review. As stated in my last comment, I will approve this PR as soon as the stock notification code is merged 🙂 
My apologies for any confusion!

File: _pages/dev/features/stock-notification.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+-  notificationoccaddon AddOn
+
+For more information, see [Stock Notification Architecture](https://help.sap.com/viewer/4c33bf189ab9409e84e589295c36d96e/1905/en-US/7afe618e1ff4437ea6a7a0c6e0c8f32b.html), [Customer Interests Architecture](https://help.sap.com/viewer/4c33bf189ab9409e84e589295c36d96e/1905/en-US/f096456e586c44a29bd833a88536855a.html) and [Notification Architecture](https://help.sap.com/viewer/4c33bf189ab9409e84e589295c36d96e/1905/en-US/b090364cfbe94c6da1b69af62f585d79.html) in the SAP Help Portal.
+
+Note that this feature works only when the notification preference feature and the customer interests feature are already working.
+
+
+## Enabling Stock Notification
+
+The stock notification feature has corresponding CMS-component data in the back end that allows you to enable or disable the feature.
```
Comment (mpern) [inline _pages/dev/features/stock-notification.md]: "corresponding CMS-componente data" -> what data? components? some flag? a special page?

**Comment by gladius-mtl** [comment]
As discussed on Slack, this PR contains too many files (5k+) that shoudn't be uploaded to github. A new PR (GH-456) was created to add the Stock Notification doc to github.

**Pull request closed without merging**